docs/content/installation/nic-images/using-the-jwt-token-docker-secret.md

```diff
@@ -150,7 +150,7 @@ If the namespace does not exist, `--create-namespace` will create it. Using `-f
 If you want to install NGINX Ingress Controller using the charts method, the following is an example of using the command line to pass the required arguments using the `set` parameter.
 
 ```shell
-helm install my-release -n nginx-ingress oci://ghcr.io/nginxinc/charts/nginx-ingress --version 1.1.2 --set controller.image.repository=private-registry.nginx.com/nginx-ic/nginx-plus-ingress --set controller.image.tag=3.4.2 --set controller.nginxplus=true --set controller.serviceAccount.imagePullSecretName=regcred
+helm install my-release -n nginx-ingress oci://ghcr.io/nginxinc/charts/nginx-ingress --version 1.1.2 --set controller.image.repository=private-registry.nginx.com/nginx-ic/nginx-plus-ingress --set controller.image.tag=3.4.3 --set controller.nginxplus=true --set controller.serviceAccount.imagePullSecretName=regcred
```

Comment:
This line still references Helm chart `--version 1.1.2` although release 3.4.3 is expected to be used via chart version 1.1.3.

Reply:
Similarly, [`charts/nginx-ingress/README.md` L90](https://github.com/nginxinc/kubernetes-ingress/pull/5128/files#diff-0ba161dd052247c6e21fb8101c5979dfa0eef87ef0632a16f572560e76ccd087L90-R90) can be observed as missing a replacement. However, [`docs/content/technical-specifications.md`](https://github.com/nginxinc/kubernetes-ingress/pull/5128/files#diff-3e4bd30977f036fa4d02dea9cbd959d65f02aff5d2792e855f9b530f410d91c7L26-R26) was updated (by hand?).

Reply:
What's strange to me with this is that `helm pull oci://ghcr.io/nginxinc/charts/nginx-ingress --untar --version 1.1.3` produces an `nginx-ingress/README.md` where `1.1.3` is mentioned.

Reply:
@sigv great catch we appreciate it. We are going to fix it.

Reply:
https://github.com/nginxinc/kubernetes-ingress/pull/5287